docs/Development.md

```diff
@@ -0,0 +1,175 @@
+<!-- TOC -->
+
+- [Have Questions](#have-questions)
+    - [Mailing Lists](#mailing-lists)
+    - [JIRA issues](#jira-issues)
+- [How to contribute](#how-to-contribute)
+    - [Becoming a committer](#becoming-a-committer)
+        - [Contributing by Helping Other Users](#contributing-by-helping-other-users)
+        - [Contributing by Testing Releases](#contributing-by-testing-releases)
+        - [Contributing by Reviewing Changes](#contributing-by-reviewing-changes)
+        - [Contributing by Documentation Changes](#contributing-by-documentation-changes)
+        - [Contributing Bug Reports](#contributing-bug-reports)
+        - [Contributing Code Changes](#contributing-code-changes)
+            - [Cloning source code](#cloning-source-code)
+            - [JIRA](#jira)
+            - [Pull Request](#pull-request)
+            - [The Review Process](#the-review-process)
+            - [Closing Your Pull Request / JIRA](#closing-your-pull-request--jira)
+            - [Code Style](#Code-Style)
+
+<!-- /TOC -->
+
+# Have Questions
+
+## Mailing Lists
+
+It is recommmended to use our mail lists to ask for help, debug issues or contributiong to the project.
```

Comment:
```suggestion
It is recommmended to use our mailing lists to ask for help, debug issues or contribute to the project.
```

Reply:
Could you create a pull request to make your changes merge into this `doc` branch?

Reply:
Glad to see you! I have fixed these reviews.